docs/virtual_machines/numa.md

```diff
@@ -0,0 +1,67 @@
+# NUMA
+
```

Comment:
Please add a small section with the scope of our "NUMA" support.

Reply:
Added a section. Hope it is better now.